cmake/platform/linux/webos.cmake

```diff
@@ -0,0 +1,9 @@
+include(${CMAKE_SOURCE_DIR}/cmake/platform/${CORE_SYSTEM_NAME}/wayland.cmake)
+
+# add wayland as platform, as we require it.
+# saves reworking other assumptions for linux windowing as the platform name.
+list(APPEND CORE_PLATFORM_NAME_LC wayland)
+
+list(APPEND PLATFORM_REQUIRED_DEPS WaylandProtocolsWebOS)
+list(APPEND PLATFORM_GLOBAL_TARGET_DEPS generate-wayland-webos-protocols)
+list(APPEND ARCH_DEFINES -DTARGET_WEBOS)
```

Comment:
this is overridden by https://github.com/xbmc/xbmc/blob/master/cmake/scripts/linux/ArchSetup.cmake#L7
we need to either move this line or adjust linux/ArchSetup to APPEND

Reply:
Adjust Linux/archsetup to a list(APPEND) imo